styles.css

```
@@ -5885,6 +5953,185 @@ img[src=''] {
+    padding: 1.5rem;
+}
+
+.email-auth-modal-header {
+    display: flex;
+    align-items: flex-start;
+    justify-content: space-between;
+    gap: 1rem;
+    margin-bottom: 1.25rem;
+}
+
+.email-auth-modal-header h3 {
```

Stylelint (17.5.0)

[error] Expected selector ".email-auth-modal-header h3" to come before selector "#queue-modal-header h3", at line 4380 (no-descending-specificity)

(no-descending-specificity)